From 191ba8a7a72a31128b3971c2671c078df3ebbc18 Mon Sep 17 00:00:00 2001
From: Martin Jansa <Martin.Jansa@gmail.com>
Date: Wed, 10 Mar 2021 22:42:25 +0100
Subject: [PATCH 2/4] service: update SettingsService path

It's installed in /usr/sbin/ not /usr/bin/ so if you have to hardcode
some path, then at least hardcode the right one.

Should fix:
WARNING: do_rootfs: QA Issue: Unable to check the binary /usr/bin/SettingsService mentioned in LS2 role files
---
 service/com.lge.settingsservice.json.prv    | 2 +-
 service/com.lge.settingsservice.json.pub    | 2 +-
 service/com.lge.settingsservice.service.prv | 2 +-
 service/com.lge.settingsservice.service.pub | 2 +-
 4 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/service/com.lge.settingsservice.json.prv b/service/com.lge.settingsservice.json.prv
index 7e54495..251befb 100644
--- a/service/com.lge.settingsservice.json.prv
+++ b/service/com.lge.settingsservice.json.prv
@@ -1,6 +1,6 @@
 {
     "role": {
-        "exeName":"/usr/bin/SettingsService",
+        "exeName":"/usr/sbin/SettingsService",
         "type": "regular",
         "allowedNames": ["com.lge.settingsservice"]
     },
diff --git a/service/com.lge.settingsservice.json.pub b/service/com.lge.settingsservice.json.pub
index 7e54495..251befb 100644
--- a/service/com.lge.settingsservice.json.pub
+++ b/service/com.lge.settingsservice.json.pub
@@ -1,6 +1,6 @@
 {
     "role": {
-        "exeName":"/usr/bin/SettingsService",
+        "exeName":"/usr/sbin/SettingsService",
         "type": "regular",
         "allowedNames": ["com.lge.settingsservice"]
     },
diff --git a/service/com.lge.settingsservice.service.prv b/service/com.lge.settingsservice.service.prv
index e740a5c..42dec3f 100644
--- a/service/com.lge.settingsservice.service.prv
+++ b/service/com.lge.settingsservice.service.prv
@@ -1,4 +1,4 @@
 [D-BUS Service]
 Name=com.lge.settingsservice
-Exec=/usr/bin/SettingsService
+Exec=/usr/sbin/SettingsService
 Type=static
diff --git a/service/com.lge.settingsservice.service.pub b/service/com.lge.settingsservice.service.pub
index e740a5c..42dec3f 100644
--- a/service/com.lge.settingsservice.service.pub
+++ b/service/com.lge.settingsservice.service.pub
@@ -1,4 +1,4 @@
 [D-BUS Service]
 Name=com.lge.settingsservice
-Exec=/usr/bin/SettingsService
+Exec=/usr/sbin/SettingsService
 Type=static
-- 
2.37.3.windows.1

